src/main/include/RobotContainer.h

```diff
@@ -97,10 +97,13 @@ class RobotContainer {
   Intake mIntake{"Intake", mIntakeRoller};
 
   //DooHickey Components
-  WPI_TalonFX mDooHickeySpinner{kDoohickey};
-  frc::DoubleSolenoid mUpPushyThang{kDoohickeyPushyThangSolenoidFoward, 
-                                    kDoohickeyPushyThangSolenoidReverse};
-  DooHickey mDooHickey{"DooHickey", mDooHickeySpinner, mUpPushyThang};
+  //WPI_TalonSRX mSpinner{kDoohickey};
+  //frc::DoubleSolenoid mUpPushyThang{kDoohickeyPushyThangSolenoidFoward, kDoohickeyPushyThangSolenoidReverse};
+  DooHickey::Components mDooHickeyComponents{
+    WPI_TalonSRX{11},
+    frc::DoubleSolenoid{0,7}
```

Comment:
There are symbols for the doohicky component ids

```suggestion
    frc::DoubleSolenoid{kDoohickeyPushyThangSolenoidFoward, kDoohickeyPushyThangSolenoidReverse}
```